assembly/std/index.ts

```diff
@@ -305,21 +432,25 @@ export function derKeys(keysSer: StaticArray<u8>): Array<StaticArray<u8>> {
 }
 
 /**
- * Converts data to base58.
+ * This function converts the given string data into a base58 formatted string.
+ *
+ * @param data - the string data to convert to base58.
  *
- * @param data -
+ * @returns the converted data as a string.
  *
  */
 export function toBase58(data: string): string {
   return env.toBase58(data);
```

Comment:
@BatiGencho This function seems to not exist in the env module is this normal ? Or should we remove this function ?

Reply:
No, I think this could be done directly on AS side. We don't need massa info for this and so passing to massa-sc-runtime will cost gas for nothing

Reply:
Since this issue is meant for documentation, should we open another one to implement this then ?

Reply:
Yes good idea.